portfolio/src/main/webapp/index.html

```diff
@@ -2,18 +2,110 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>Kayla Leung Portfolio</title>
+    <title>Kayla Leung</title>
+
     <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="font.css">
+
     <script src="script.js"></script>
+    <script src="https://kit.fontawesome.com/b99e675b6e.js"></script>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    
+
   </head>
   <body>
-    <div id="content">
-      <h1>Kayla Leung</h1>
-      <p>This is my portfolio.</p>
-      <img src="images/kaylaleung.jpeg"/>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+
+    <!-- Toggle Side View -->
+    <input type="checkbox" name="">
```

Comment:
It's cool to be able to implement your slide in/out menu with JavaScript, but there are a couple problems. One is that screen readers get confused by a random checkbox that's not in a form and isn't attached to anything. Another is that sibling selectors (the CSS with the ~, which this approach depends on) have poor performance to the point where Google internally bans them. But this is more FYI, I'm not going to ask you to change it.